.travis.yml

```diff
@@ -70,7 +72,8 @@ script: .ci/test_script.sh
 
 after_success:
   # upload coverage report to coveralls.io
-  - if [ "$TEST_TYPE" == "tests" ]; then coveralls; fi
+  # Do for all travis runs, even if there is no coverage to report
+  COVERALLS_PARALLEL=true coveralls
```

Comment:
@giovannipizzi You are missing a dash here

Reply:
Thanks! I just fixed this, but I'm not sure this is the cause of the problem, because the coveralls run was running, see e.g. https://travis-ci.org/aiidateam/aiida_core/jobs/430422950#L2154

Also, in #1978 the message gets added. Let's see what happens here in the meantime.